README.md

```diff
@@ -155,6 +157,19 @@ If the barrier method returns `false`, message will be returned into the queue f
 
 > **_NOTE:_**  See [SqsPermissionConsumer.ts](./packages/sns/test/consumers/SnsSqsPermissionConsumer.ts) for a practical example.
 
+### Dead Letter Queue
+A dead letter queue is a queue where messages that cannot be processed successfully are stored. It serves as a holding area for messages that have encountered errors or exceptions during processing, allowing developers to review and handle them later.
+To create a dead letter queue, you need to specify the `deadLetterQueue` parameter within options on the consumer configuration. The parameter should contain the following fields:
+- `creationConfig`: configuration for queue to create, if one does not exist. Should not be specified together with the `locatorConfig`
+- `locatorConfig`: configuration for resolving existing queue. Should not be specified together with the `creationConfig`
+- `redrivePolicy`: an object that contains the following fields:
```

Comment:
this is very SQS-specific, I wonder if we should have a higher-level concept on top of it

Reply:
Will make it more generic once we address the comment below, thanks Igor 🙇

Reply:
Added a note a top explaining DLQ support, so I guess it is fine to keep this coupled to SQS for now